pkg/pod/container.go

```diff
@@ -235,6 +235,29 @@ func (builder *ContainerBuilder) WithEnvVar(name, value string) *ContainerBuilde
 	return builder
 }
 
+// WithVolumeMounts configures pod volumes to mount inside the container.
+func (builder *ContainerBuilder) WithVolumeMounts(volMounts []v1.VolumeMount) *ContainerBuilder {
+	glog.V(100).Infof("Adding VolumeMount(s) to the %q container's definition", builder.definition.Name)
+
+	if len(volMounts) == 0 {
+		glog.V(100).Infof("Container's VolumeMounts list is empty")
+
+		builder.errorMsg = "container's volume mount list is empty"
+	}
+
+	if builder.errorMsg != "" {
+		return builder
+	}
+
+	if builder.definition.VolumeMounts != nil {
```

Comment:
this one could be as @ElaiShalevRH said:

`builder.definition.VolumeMounts = append(builder.definition.VolumeMounts, volMounts...)`

Reply:
thanks, updated